src/shared/components/plots/WaterfallPlot.tsx

```diff
@@ -72,6 +72,10 @@ const limitValueAppearanceSupplement = {
     strokeOpacity: 1,
     size: 3
 };
+const plotTitleStyles = {
+    textAlign: 'center' as 'center',
+    margin: '20px 0 10px 0'
+};
```

Comment:
Better to put this in a `style.scss` file in the src/shared/components/plots, import it and apply via classname.